plugin/init.lua

```diff
@@ -227,6 +227,7 @@ function pub.switch_to_prev_workspace()
 			}),
 			pane
 		)
+		wezterm.emit("smart_workspace_switcher.workspace_switcher.switched_to_prev", window, pane)
```

Comment:
It would be nice to also have the new workspace emitted like so:
```suggestion
		wezterm.emit("smart_workspace_switcher.workspace_switcher.switched_to_prev", window, pane, previous_workspace)
```